tests/phpunit/api/v3/MailingContactTest.php

```diff
@@ -33,11 +33,11 @@ public function setUp(): void {
       'contact_type' => 'Individual',
       'last_name' => 'xyz1',
     ];
-    $this->_contact = $this->callAPISuccess("contact", "create", $params);
+    $this->contact = $this->callAPISuccess("contact", "create", $params);
```

Comment:
If I had been touching this line I would also have standardised the casing & use of quotes - ie 
`$this->callAPISuccess('Contact', 'create', $params);`